account_easy_reconcile/easy_reconcile_history.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Guewen Baconnier
+#    Copyright 2012 Camptocamp SA
+#    Copyright 2015 Camptocamp SA (Damien Crier)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, api, fields, _
+
+
+class EasyReconcileHistory(models.Model):
+    """ Store an history of the runs per profile
+    Each history stores the list of reconciliations done"""
+
+    _name = 'easy.reconcile.history'
+    _rec_name = 'easy_reconcile_id'
+    _order = 'date DESC'
+
+    @api.one
+    @api.depends('reconcile_ids', 'reconcile_partial_ids')
+    def _reconcile_line_ids(self):
```

Comment:
Could be renamed `_compute_reconcile_line_ids` or `_get_reconcile_line_ids` for clarity